homeassistant/components/airzone/select.py

```diff
@@ -33,6 +38,9 @@ class AirzoneSelectDescription(SelectEntityDescription):
 
     api_param: str
     options_dict: dict[str, int]
+    options_fn: Callable[[dict[str, Any], dict[str, int]], dict[str, int]] = (
```

Comment:
Why does this actually return a dict and not a list? the options will be a `list` in the end

Reply:
Nice catch, fixed in 2e573a1d1e3e33d5ba04a5df2a7a5d965899df51.
It was returning a dict because at first I was using it for `values_dict` too, but I changed that to use `description.options_dict` instead.